Empire/Empire.h

```diff
@@ -297,6 +297,7 @@ class FO_COMMON_API Empire {
     void SetReady(bool ready);               ///< Marks this empire with readiness status
     void AutoTurnSetReady();                 ///< Decreases auto-turn counter and set empire ready if not expired or set unready
     void SetAutoTurn(int turns_count);       ///< Set auto-turn counter
+    void SetLastTurnReceived(int last_turn_received) noexcept; ///< Set last turn received
```

Comment:
is this ever called? or you just want to add the function and data without using it?

Reply:
Forgot to put call when reorganized it.